cmake/macros/rv_make_std_lib_name.cmake

```diff
@@ -0,0 +1,60 @@
+
+MACRO(RV_MAKE_STANDARD_LIB_NAME base_name version lib_type debug_postfix)
+
+  IF(CMAKE_BUILD_TYPE MATCHES "^Debug$")
+    SET(_debug_postfix "${debug_postfix}")
```

Comment:
cmake-format has not been run on this file (I triple-checked because I was dans le champs for the other one)